apps/google-analytics-4/frontend/src/components/main-app/AnalyticsApp/AnalyticsApp.tsx

```diff
@@ -46,6 +46,7 @@ const AnalyticsApp = (props: Props) => {
         const reportData = await api.runReports(reportRequestParams);
         setRunReportResponse(reportData);
       } catch (e) {
+        console.error(e);
```

Comment:
nit: Double checking, was this intended?

Reply:
good eye, was using to test. Will remove :)